cinema/views.py

```diff
@@ -21,24 +24,62 @@
 )
 
 
-class GenreViewSet(viewsets.ModelViewSet):
+class IsAdminOrIfAuthenticatedReadOnly(BasePermission):
```

Comment:
it's better to move it to a separate module.
What if you need 10 more permissions? Your views would be enormous